ref/config/serverConfiguration.adoc

```diff
@@ -221,37 +175,28 @@ operator can be one of `+`, `-`, `*`, `/` for example:
 <variable name="threeagain" value="${six/two}" />
 ----
 
-There are a number of predefined variables:
+The following predefined variables are supported:
 
-* `wlp.install.dir` - the location where the Liberty runtime is installed.
-* `wlp.server.name` - the name of the server.
-* `wlp.user.dir` - the location of the `usr` folder. Defaults to
+* `wlp.install.dir`: the location where the Open Liberty runtime is installed.
+* `wlp.server.name`: the name of the server.
+* `wlp.user.dir`: the location of the `usr` folder. The default is
   `${wlp.install.dir}/usr`.
-* `shared.app.dir` - the location of shared applications. Defaults to
+* `shared.app.dir`: the location of shared applications. The default is
   `${wlp.user.dir}/shared/apps`.
-* `shared.config.dir` - the location of shared configuration files. Defaults to
+* `shared.config.dir`: the directory that contains the server config. The default is
   `${wlp.user.dir}/shared/config`.
-* `shared.resource.dir` - the location of shared resource files. Defaults to
+* `shared.resource.dir`: the location of shared resource files. The default is
   `${wlp.user.dir}/shared/resources`.
-* `server.config.dir` - the directory that server config is stored in.
-  Defaults to `${wlp.user.dir}/servers/${wlp.server.name}`.
-* `server.output.dir` - the directory that the server writes the workarea, logs and
-  other runtime generated files to. Defaults to `${server.config.dir}`.
-
-=== Config merging
-The config can be made up of multiple files so it is possible, perhaps
-even likely, that two files provide the same config. In these
-situations the server config is merged using a set of simple rules. In
-{projectName}, config is separated into singleton and factory
-config. Merging works differently for the two. Singleton config
-is used when configuring a single thing (e.g. logging); factory config is
-used when it is valid to configure multiple things, (e.g. an application or a
-data source).
-
-==== Merging singleton config
-
-For singleton config elements the config is merged. If two
-elements exist with different attributes both attributes are used. For example:
+* `server.config.dir`: the directory that server config is stored in.
+  The default is `${wlp.user.dir}/servers/${wlp.server.name}`.
+* `server.output.dir`: the directory where the server writes the workarea, logs, and
+  other runtime-generated files. The default is `${server.config.dir}`.
+
+== Configuration merging
+Since the configuration can consist of multiple files, it is possible that two files provide the same configuration. In these situations, the server config is merged according to a set of simple rules. In {projectName}, configuration is separated into Singleton and factory configuration, each of which has its own rules for merging. Singleton config is used to configure a single element (for example, logging). Factory config is used to configure multiple entities, such as an entire application or data source.
```

Comment:
Singleton isn't a proper noun so should be a lower case s.

More inconsistent use of config vs configuration

Reply:
Updated